libreyolo/models/fomo/trainer.py

```
@@ -0,0 +1,577 @@
+                    f"grid_precision={raw_metrics.get('metrics/grid_precision', 0.0):.4f} | "
+                    f"grid_recall={raw_metrics.get('metrics/grid_recall', 0.0):.4f} | "
+                    f"MeanDist={raw_metrics.get('metrics/grid_mean_distance', 0.0):.3f} | "
+                    f"thresh={raw_metrics.get('decode/threshold', 0.0):.2f} | "
+                    f"nms_r={raw_metrics.get('decode/nms_radius', 0.0):.1f} | "
+                    f"TP={raw_metrics.get('metrics/grid_TP', 0.0):.1f} FP={raw_metrics.get('metrics/grid_FP', 0.0):.1f} FN={raw_metrics.get('metrics/grid_FN', 0.0):.1f} | "
+                    f"LR={current_lr:.6f}"
+                )
+
+            return metrics
+
+        except Exception as exc:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)